scripts/find-low-dpi-images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Created by Sam Gleske (@samrocketman on GitHub)
+# MIT licensed
```

Comment:
This should probably be GPL licensed like everything else in ES.

Reply:
I typically license my code MIT which is compatible with GPL3.